ksql-engine/src/test/java/io/confluent/ksql/function/udf/datetime/TimestampToStringTest.java

```diff
@@ -46,6 +46,30 @@ public void shouldCovertTimestampToString() {
     assertThat(result, is(expectedResult));
   }
 
+  @Test
+  public void testUTCTimeZone() {
+    // When:
+    final Object result = udf.evaluate(1534353043000L, "yyyy-MM-dd HH:mm:ss", "UTC");
```

Comment:
Can you add the zone to the format string?

Reply:
how in the format string?

Reply:
Right now, this solution is designed to don't specify the timeZone in the format string.

What is the desired behavior?
1. The user can specify the TimeZone in the FormatString (in this case, what happens when the user specifies additionally a TimeZone field different from the local in FormatString?)
2. The user just can specify the TimeZone ONLY in the TimeZone field (not in the formatString).

Reply:
You can include a pattern letter in the format string that writes the zone out to the result:
See z, Z, and X here: https://docs.oracle.com/javase/7/docs/api/java/text/SimpleDateFormat.html

Reply:
@rodesai Ready. Yes, you can. I've added some testcases

Reply:
I meant in this test and testPSTTimeZone. I just want to see that the same time stamp gets printed with different date times and zones.

Reply:
@rodesai Ready. Yes it is. I've added more testcases.